### PR TITLE
updates for automation (and release)

### PR DESCRIPTION
**Pull request by andy-neuma**
SUMMARY:
* enable python 3.9 and 3.11 in "remote push"
* update "nightly" to run for all pythons
* for python 3.8, 3.9. 3.11 just enable minimal testing for now
* update install commands used in automation to include https://pypi.neuralmagic.com/simple
* update tmp skip list to skip almost everything

NOTES: adjusting the skip lists will be completed in an upcoming PR. i'd like to get these updates in place so we can nominally start pushing a NIGHTLY

TEST PLAN:
runs on remote push

### CODE REVIEW

File: .github/workflows/remote-push.yml [hunk 1]
```diff
@@ -20,7 +20,23 @@ jobs:
             test_label_solo: aws-avx2-32G-a10g-24G
             test_label_multi: ignore
             test_timeout: 480
-            test_skip_list: neuralmagic/tests/skip-for-remote-push.txt
+            test_skip_list: neuralmagic/tests/skip-for-remote-push-tmp.txt
```
Comment (dbarbuzzi) [inline .github/workflows/remote-push.yml:23]: Seems like the py310 job is still using the non-tmp version of this file.

File: neuralmagic/tests/skip-for-remote-push-tmp.txt
Comment (dbarbuzzi) [inline neuralmagic/tests/skip-for-remote-push-tmp.txt:1]: At some point in the future, maybe we can sort these files’ lines alphabetically?

File: .github/workflows/nightly.yml [hunk 2]
```diff
@@ -17,7 +17,43 @@ on:
...
+        uses: ./.github/workflows/build-test.yml
+        with:
+            wf_category: NIGHTLY
+            python: 3.8.17
+            gitref: ${{ github.ref }}
+
+            test_label_solo: aws-avx2-32G-a10g-24G
+            test_label_multi: aws-avx2-192G-4-a10g-96G
+            test_timeout: 480
+            test_skip_list: neuralmagic/tests/skip-for-remote-push-tmp.txt
```
Comment (dbarbuzzi) [inline .github/workflows/nightly.yml:30]: Again, the py310 job is not using the `-tmp` version of this file.

**Review by dbarbuzzi — APPROVED** [review]
A couple of comments about differing versions of the skip list file but probably non-blocking.

File: neuralmagic/tests/skip-for-remote-push-tmp.txt
Comment (andy-neuma) [reply neuralmagic/tests/skip-for-remote-push-tmp.txt:1]: yeah, or have something better :)

File: .github/workflows/nightly.yml (hunk 2 above)
Comment (andy-neuma) [reply .github/workflows/nightly.yml:30]: same comment as above.

**Pull request merged**